shelley/chain-and-ledger/formal-spec/incentives.tex

```diff
@@ -0,0 +1,572 @@
+The cost is subtracted from the total rewards that are earned by the pool before any rewards are distributed.
+These parameters are advertised by the stakepool and used as part of the public ranking scheme.
+
+\subsubsection*{The \ada{} that is controlled by the Stakepool}
+
+\begin{figure}[h!]
+\begin{center}
+\begin{tabular}{||l|p{6cm}|l||}
+  \hline \hline
+\textbf{Parameter} & \textbf{Description} & \textbf{Calculated as} \\\hline
+$SP_p$ & \ada{} that is pledged to the Stakepool by the Owner(s) & \\\hline
+$SP_d$ & \ada{} that is delegated to the Stakepool & \\\hline
+$SP_T$ & All \ada{} that is controlled by the Stakepool & $SP_p + SP_d$ \\\hline
+$SP_F$ & Fraction of the \ada{} in circulation that is controlled by the Stakepool & {\large $\frac{SP_T}{A_c}$} \\\hline
+\end{tabular}
+\end{center}
+\caption{The \ada{} that is controlled by a StakePool.}
+\end{figure}
+
+\noindent
+The rewards that are received by the Stakepool will be proportional to the amount of \ada{} that it controls.
+This is calculated as the sum of the \ada{} that is \emph{pledged} to the Stakepool by the owners(s) plus
+the additional \ada{} that is \emph{delegated} to the Stakepool.  An owner may \emph{delegate} to a Stakepool if
+they wish as well as \emph{pledging} to it, but would receive lower rewards.  In return for receiving
+higher rewards, \emph{pledging} incurs higher risk, requiring owners to \emph{trust} each other.
+
+
+\section{The Simplified Incentives Scheme (Shelley Incentived Testnet)}
+\label{sec:testnet}
+
+The simplified scheme calculates rewards based on the total number of blocks that each stake pool produces,
+and distributes a fixed amount of \ada{} per epoch.
+Since \cardano{} is based on \emph{proof of stake}, then on average, each stake pool will produce
+rewards that are proportional to the stake that it holds.  So, if e.g. a pool holds 1\% of the total
+\ada{} in circulation, then it will receive, on average, 1\% of the total rewards that are allocated to all the
+stakepools.
+
+\begin{figure}[h!]
+\begin{center}
+\begin{tabular}{||l|l|p{6cm}|l||}
+  \hline \hline
+\textbf{Parameter} & \textbf{Expected Value} & \textbf{Description} & \textbf{Calculated as} \\\hline
+\emph{P} & 20\%-50\% & Participation Rate in the Testnet & \\\hline
+$A_i$ & \ADA{15.5bn} @ $P=50\%$ & The \ada{} circulating in the Testnet & $P \times A_c$ \\\hline
+\end{tabular}
+\end{center}
+\caption{\ada{} in Circulation in the Incentivised Testnet.}
+\end{figure}
+
+\noindent
+The total \ada{} that is circulating in the Testnet is derived from the \emph{participation rate} (the percentage of the total \ada{} in
+circulation that is available to be staked in the Incentivised Testnet). This will be determined at the date when the
+Testnet launches and will then remain fixed\khcomment{Check - no \ada{} can be transferred later -
+affects the calculations/reward?}.  It is assumed that this is likely to be 20\%-50\% of the total \ada{} in
+circulation.
```

Comment:
Maybe add a comment which "type" of ADA cannot be staked.

Reply:
Any ADA in circulation can be staked.  You just need the right key.